src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-tutorials/opentelemetry-tutorial-java.mdx

```diff
@@ -0,0 +1,1132 @@
+---
+title: 'OpenTelemetry tutorials: Instrument a sample Java app'
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: You can set up OpenTelemetry to monitor a simple Java app.
```

Comment:
Might rephrase this. 1) I try to work in New Relic and what it can do for you in the metadescrip. 2) it sounds like you can use OTel to only monitor a simple app, not a complex app; not clear we're talking about an example test app (might have been typo for 'sample')